src/PrefsManager.cpp

```diff
@@ -307,6 +307,7 @@ PrefsManager::PrefsManager() :
 	m_custom_songs_max_megabytes("CustomSongsMaxMegabytes", 5.f),
 
 	/* Debug: */
+	m_bDebugMenuEnabled	("DebugMenuEnabled", true),
```

Comment:
I would recommend changing this line to:

`m_bDebugMenuEnabled ("DebugMenuEnabled", true, nullptr, PreferenceType::Immutable)`

To prevent things like custom lua charts from changing this value programmatically. That way to actually disable it, an operator has to set the settings to whatever, close the game, and manually update set the value in Preferences.ini, which I think is a safe approach to guarantee it can't be changed from the theme or elsewhere.

Reply:
Thanks for the suggestion! I've implemented this, and added some better commentary to the fake class/DECLARE_ONE section.